src/chunk_index.c

```diff
@@ -907,6 +912,59 @@ chunk_index_tuple_rename(TupleInfo *ti, void *data)
 	return SCAN_DONE;
 }
 
+static ScanTupleResult
+chunk_index_tuple_adjust_meta(TupleInfo *ti, void *data)
```

Comment:
This function is very similar to `chunk_index_tuple_rename`. It is unclear why we need both. At the very least they should be able to share most code. See related comment below.